packages/editor/src/browser/editor.ts

```diff
@@ -51,6 +52,8 @@ export interface TextEditor extends Disposable, TextEditorSelection {
      */
     resizeToFit(): void;
     setSize(size: Dimension): void;
+
+    setDecorations(key: string, decorations: DecorationOptions[]): void;
```

Comment:
how can one remove decorations? Should not it return `Disposable`

Reply:
they are removed by `setDecorations(key, [])`. I'll add a comment.